WTWSMS/common/minor_titles/WtWSMS_minor_titles.txt

```diff
@@ -132,20 +132,21 @@ title_sigecempa = {
 }
 
 title_court_celtic_poet = { # Celtic Poet/Bard
-	dignity = 0.25
+	dignity = 1
 	realm_in_name = yes
 	grant_limit = 1
 	opinion_effect = 30
 	
 	monthly_salary = 0.02
-	monthly_prestige = 0.01
+	monthly_prestige = 0.1
```

Comment:
Disagreed, it is a honorific title which would only be used intermittently in the case of the court poets.